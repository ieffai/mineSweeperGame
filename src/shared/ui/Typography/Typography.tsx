import { default as BaseTypography, type TypographyProps as BaseTypographyProps } from '@mui/material/Typography';

export type TypographyProps = BaseTypographyProps;
export const Typography = BaseTypography;

export default Typography;
