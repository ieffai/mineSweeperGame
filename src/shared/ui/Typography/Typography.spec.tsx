import { render } from '@testing-library/react';

import Typography from './Typography';

describe('Typography', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<Typography />);
    expect(asFragment()).toMatchSnapshot();
  });
});
