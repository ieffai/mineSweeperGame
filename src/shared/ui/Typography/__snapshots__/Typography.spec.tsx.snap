// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Typography > should render successfully 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
  />
</DocumentFragment>
`;
